opencti-platform/opencti-front/src/private/components/analysis/groupings/GroupingKnowledgeGraphBar.js

```diff
@@ -220,8 +213,15 @@ class GroupingKnowledgeGraphBar extends Component {
     if (
       this.props.numberOfSelectedNodes === 1
       && !this.props.selectedNodes[0].parent_types.includes('basic-relationship')
+      && !this.props.selectedNodes[0].parent_types.includes('Stix-Cyber-Observable')
```

Comment:
We can have directly this ?

if( this.props.selectedNodes[0].parent_types.includes('Stix-Domain-Object') ) {
         this.setState({ openEditObservable: true });
} else if ( this.props.selectedNodes[0].parent_types.includes('Stix-Cyber-Observable') ) {
         this.setState({ openEditObservable: true });
}

Reply:
are you sure for "if( this.props.selectedNodes[0].parent_types.includes('Stix-Domain-Object') ) {"
(because it was writen "!this.props.selectedNodes[0].parent_types.includes('basic-relationship')" at first...

Reply:
No I'm not sure at all.
It can be test or let it like this